mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_notification_config_test.go

```diff
@@ -0,0 +1,140 @@
+package securitycenter_test
```

Comment:
The previous failures were probably related to this package. I think it needs to be `securitycenterv2_test`

Reply:
addressed